verilog/formatting/tree_unwrapper.cc

```diff
@@ -892,6 +950,31 @@ void TreeUnwrapper::Visit(const verible::SyntaxTreeNode& node) {
       // else close-out current token partition?
       break;
     }
+    case NodeEnum::kIfClause: {
+      if (Context().DirectParentsAre(
+              {NodeEnum::kConditionalStatement, NodeEnum::kElseBody}) &&
```

Comment:
Would it make more sense to perform these transformations as a case of `kElseClause` and `kConditionalStatement` (looking downward instead of up the context stack)?

Reply:
I've rewritten it.